lib/improver/nbhood.py

```diff
@@ -170,6 +172,36 @@ def get_neighbourhood_width_in_grid_cells(
             )
         return grid_cells_x, grid_cells_y
 
+    @staticmethod
+    def adjust_nsize_for_ens(ens_factor, num_ens, width_in_km):
+        """
+        Adjust neighbourhood size according to ensemble size.
+
+        Parameters
+        ----------
+        ens_factor : float
+            Defined Factor with which to adjust the neighbour size
+            If ens_factor = 1.0 this essentially conservers ensemble
+            members if every grid square is considered to be the
+            equivalent of an ensemble member.
```

Comment:
spellings! (sorry)
"conserves"
Use "neighbourhood size" instead of "neighbour size"
See also comment in class NeighbourhoodProcessing.